### PR TITLE
Use dep vendoring tool

### PR DESCRIPTION
**Pull request by sterchelen**
Linked to #1

### CODE REVIEW

**Comment by itscaro** [comment]
@sterchelen Could you sign-off you commit and re-push? `git commit -s --amend`

**Comment by sterchelen** [comment]
@itscaro signed-off ;)

File: Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
```
Comment (dungeonmaster18) [inline Gopkg.toml]: Add faas as constraint.

**Comment by dungeonmaster18** [comment]
Also please exclude those vendor files. Only Gopkg.toml and Gopkg.lock. Sorry for not informing thiis before. Once done please ping me.

**Comment by sterchelen** [comment]
@dungeonmaster18 Done !

**Comment by alexellis** [comment]
LGTM. @dungeonmaster18 please can you checkout this PR and confirm it works?

**Comment by alexellis** [comment]
@sterchelen thanks for the PR. Please provide details on how you have tested this work.

**Comment by sterchelen** [comment]
Sorry @alexellis ... Preparing the coming of my baby :) !

I just ran make test.

**Comment by alexellis** [comment]
Congratulations :tada: 

How did it look? did it work?

**Comment by sterchelen** [comment]
It lookis good ! All tests passed.

**Comment by alexellis** [comment]
Great :+1: thanks

**Comment by alexellis** [comment]
Hi @sterchelen since I merged this change the tests appear to fail on every build.. PTAL:

https://travis-ci.org/openfaas/faas/builds/349273728#L1086

cc @dungeonmaster18

**Comment by dungeonmaster18** [comment]
Hey I am on it.
What does time is for? Because I am getting an error due to it.
 gateway_url=http://localhost:8080/ time go test ./tests -v

**Comment by dungeonmaster18** [comment]
Hey @alexellis fixed the test fail and created a pr have a look.

**Pull request merged**